core/frontend/src/tile/ImdlReader.ts

```diff
@@ -223,15 +223,31 @@ export class ImdlReader extends GltfReader {
   }
 
   private async readNamedTexture(namedTex: any, name: string): Promise<RenderTexture | undefined> {
-    const bufferViewId = JsonUtils.asString(namedTex.bufferView);
-    const bufferViewJson = 0 !== bufferViewId.length ? this._bufferViews[bufferViewId] : undefined;
-    if (undefined === bufferViewJson)
-      return undefined;
+    let texBytes: Uint8Array | undefined;
 
-    const byteOffset = JsonUtils.asInt(bufferViewJson.byteOffset);
-    const byteLength = JsonUtils.asInt(bufferViewJson.byteLength);
-    if (0 === byteLength)
-      return undefined;
+    // If the texture has a valid Id64 name, it is valid to request it directly from the backend.
+    // If the texture does not have a valid Id64 name, it is not valid to request it directly from the backend.
+    // In that case, we instead retrieve the texture directly from the tile.
+    // Furthermore, if external textures are disabled, we retrieve the texture directly from the tile in both cases.
+
+    const useExternalTexture = Id64.isValidId64(name) && IModelApp.tileAdmin.enableExternalTextures;
+    if (useExternalTexture) {
+      texBytes = await this._iModel.getTextureImage(name);

```

Comment:
We talked about wanting to avoid awaiting texture loads while loading tile contents. The way you currently have it, if a tile contains 15 textures, we will make 15 requests in sequence, and the tile will not finish loading until all of them complete. Back in the day, didn't we use a placeholder 1-pixel image until the actual image became available?

Reply:
I think we did do something like that previously (the 1x1 image).  I'll work on trying to get this going.

Reply:
This has been added.